notebooks/dgi_annoy.py

```diff
@@ -0,0 +1,190 @@
+'''
```

Comment:
Some of this code has already been reviewed in this [PR](https://github.com/alphagov/govuk-content-similarity/pull/5).